chalicelib/app_utils.py

```diff
@@ -240,11 +273,15 @@ def trim_output(output, max_size=100000):
 
     Takes in the non-json formatted version of the fields. For now, just use this for /view/.
     """
-    formatted = json.dumps(output, indent=4)
-    if len(formatted) > max_size:
-        return ''.join([formatted[:max_size], '\n\n... Output truncated ...'])
-    else:
-        return formatted
+    # formatted = json.dumps(output, indent=4)
+    # if len(formatted) > max_size:
+    #     return ''.join([formatted[:max_size], '\n\n... Output truncated ...'])
+    # else:
+    #     return formatted
+    size = get_size(output)
+    if size > max_size:
+        return 'Output too large to provide on main page - see check result directly'
```

Comment:
If you make this a named constant, the test file can test for that named constant and then you can still do rewordings, typo corrections, etc.  

(Random trivia: Before the dawn of time, error messages used to just be strings so could not be fixed once people started using them. One of the big excitements of object-oriented error messages wasn't just the class-system but was the indirection between an error "object" and an error "message". The same need exists here even though you are not raising an error .)